modules/core/src/main/scala/org/scalasteward/core/nurture/NurtureAlg.scala

```diff
@@ -42,7 +41,6 @@ final class NurtureAlg[F[_]](config: Config)(implicit
     gitAlg: GitAlg[F],
     logger: Logger[F],
     pullRequestRepository: PullRequestRepository[F],
-    scalafixMigrationsFinder: ScalafixMigrationsFinder,
```

Comment:
That we can get rid of `ScalafixMigrationsFinder` in `NurtureAlg` is what I like most about this change.